libs/feedback/src/lib/feedback-widget/feedback-widget.component.ts

```diff
@@ -29,12 +30,7 @@ export class FeedbackWidgetComponent implements OnInit, OnDestroy {
   ) {
     this.router.events
       .pipe(takeUntil(this.destroy))
-      .subscribe(
-        () =>
-          (this.messages$ = this.feedbackService.getMessages(
-            this.activatedRoute
-          ))
-      );
+      .subscribe(() => this.messages$ = this.feedbackService.getMessages(this.activatedRoute));
```

Comment:
I know it's not your code, but can we either unsubscribe here, or avoid subscription?
Alternatively just leave a TODO